packages/react/src/components/FormControl/FormControl.tsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import MuiFormControl, {FormControlProps as MuiFormControlProps} from '@mui/material/FormControl';
+import clsx from 'clsx';
+import {ElementType, FC, ReactElement} from 'react';
+import {WithWrapperProps} from '../../models';
+import {composeComponentDisplayName} from '../../utils';
+import './form-control.scss';
+
+export type FormControlProps<C extends ElementType = ElementType> = {
```

Comment:
What's the purpose of it?`ElementType = ElementType`

Reply:
This is done to use the `component` prop in wrapped MUI components. i.e. to make it polymorphic. We use this so we don't have to provide an element type to the component when using the `component` prop. And since we don't always use that prop the default type of the component will remain as `ElementType`.